securedrop/journalist.py

```diff
@@ -186,8 +188,13 @@ def admin_add_user():
                 db_session.commit()
             except InvalidPasswordLength:
                 form_valid = False
-                flash("Your password is too long (maximum length {} characters)".format(
-                        Journalist.MAX_PASSWORD_LEN), "error")
+                flash(
+                    "Your password is too long "
+                    "(maximum length {} characters)".format(
+                        Journalist.MAX_PASSWORD_LEN
+                    ),
+                    "error"
+                )
```

Comment:
In this one it's not just for compactness--I think the spacing you have here actually hurts readability.